index.html

```diff
@@ -1,11 +1,15 @@
 <!DOCTYPE html>
 <html lang="es">
+
 <head>
     <meta charset="UTF-8">
     <title>Formulario de Compra</title>
```

Comment:
SHOULD: Se debe tener en cuenta que a futuro el titulo debe cambiarse a mayuscula